common/src/common.py

```diff
@@ -73,4 +73,19 @@ def diff_and_save(data, fname, verbose=True):
   with open(fname, 'w') as f:
     json.dump(data, f)
 
-  return diff
\ No newline at end of file
+  return diff
+
+def get_clerk_data_default_entry():
+  return {
```

Comment:
@tianhuil Is the way that I'm setting these default field values correct? Based on the readme / tests it seems like everything except Locale should default to None, but you mentioned "emails" should default to [].

Reply:
You don't have to do this.  The fields are all of the form `email?: string[]` which in typescript means they can just be empty (no field defined).  I guess technically, the only required field is `locale`.  Using `None` is also fine but leaving it blank is fine.

Sorry that wasn't clear.  I'll add something about that.

I think we can nix this function.

Reply:
makes sense, thanks!